### ISSUE TITLE
I am getting below exception on composer update after adding vinelab?

### ISSUE THREAD

**Issue by hiddenl00p** [Vinelab/cdn#54]
[RuntimeException]                                                                                                                                  
  Could not scan for classes inside "/home/vagrant/code/dev/vendor/vinelab/cdn/tests/TestCase.php" which does not appear to be a file nor a f  
  older

**Comment by olimorris** [comment]
Yes I'm getting this too!
```
[RuntimeException]                                                                                                                                                     
  Could not scan for classes inside "/home/forge/default/releases/20150524121335/vendor/vinelab/cdn/tests/TestCase.php" which does not appear to be a file nor a folder
```
When you install the package. The 'tests' directory is not included with it which is causing the error.

Guys - This is a breaking issue!

**Comment by Mahmoudz** [comment]
Fix in #621adbb0bcedd02c8f3b2ea3b122b4a9ecb31011 
I update the git attributes config file to keep the /tests directory when export-ignore is used with git.
The `.gitattributes` file is telling git to ignore the `/tests` directory when downloading the package via composer in your project. Since the `/test` directory is autoloaded by the `composer.josn` file it is causing this RuntimeException.
